README.md

```diff
@@ -1,11 +1,25 @@
 # Arbiter [![Build Status](https://travis-ci.org/jspahrsummers/Arbiter.svg?branch=master)](https://travis-ci.org/jspahrsummers/Arbiter)
 
-Arbiter is a cross-platform C library<sup>1</sup> which implements the basic behaviors needed by any decentralized dependency manager or package manager, without being coupled to any particular use case, so that many more specific tools can be built on top.
+Arbiter is a cross-platform C library<sup>1</sup> which implements the baseline functionality that should be expected of any dependency manager or package manager, without being coupled to any particular use case, so that many more specific tools can be built on top.
 
 In other words, **Arbiter does not prescribe any one user experience**—it just tries to solve those backend concerns which are common to all dependency managers.
 
 _<sup>1</sup> Note that Arbiter is actually implemented in C++14, but currently only exposes a plain C API to minimize surface area and maximize interoperability._
 
+## Motivation
+
+Arbiter’s main contributors originally built [Carthage](https://github.com/Carthage/Carthage), a dependency manager for iOS and OS X frameworks. Carthage introduced some novel ideas which aren’t commonly found in other dependency managers, like a focus on [decentralization](#lazy-decentralized-dependency-resolution). Unfortunately, Carthage is fairly coupled to Apple’s development tools and process.
```

Comment:
Playing mostly the same role I did with Carthage! But I won't be maintaining this one. 😜